OpenSourceOrg/dotOrg
# Make press-mention's title linkable in the single page

#### Changes proposed in this Pull Request

* Made the main title of the press-mention's single page clickable 

#### Testing instructions

* Edit a press-mention post from the backend and make sure it has the "Article URL" custom field filled
* Open the page on the Frontend and confirm that its title is linkable and it opens a new window to the article URL when clicked.

Fixes #10
